runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java

```diff
@@ -197,76 +201,67 @@ private void findExamineItem(PendingExamine pendingExamine)
-					itemId = widgetItem.getItemId();
+			else if (pendingExamine.getType() == ExamineType.ITEM_BANK_EQ) {
+				if (WidgetInfo.EQUIPMENT.getGroupId() == widgetGroup) {
+					Widget widgetItem = widget.getChild(1);
+					if (widgetItem != null) {
+						quantity = widgetItem.getItemQuantity();
+						itemId = widgetItem.getItemId();
+					}
+				} else if (WidgetInfo.SMITHING_INVENTORY_ITEMS_CONTAINER.getGroupId() == widgetGroup) {
+					Widget widgetItem = widget.getChild(2);
+					if (widgetItem != null) {
+						quantity = widgetItem.getItemQuantity();
+						itemId = widgetItem.getItemId();
+					}
+				} else if (WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getGroupId() == widgetGroup
+						|| WidgetInfo.RUNE_POUCH_ITEM_CONTAINER.getGroupId() == widgetGroup) {
+					Widget widgetItem = widget.getChild(pendingExamine.getActionParam());
+					if (widgetItem != null) {
+						quantity = widgetItem.getItemQuantity();
+						itemId = widgetItem.getItemId();
+					}
+				} else if (WidgetInfo.BANK_ITEM_CONTAINER.getGroupId() == widgetGroup) {
+					Widget[] children = widget.getDynamicChildren();
+					if (pendingExamine.getActionParam() < children.length) {
+						Widget widgetItem = children[pendingExamine.getActionParam()];
+						quantity = widgetItem.getItemQuantity();
+						itemId = widgetItem.getItemId();
+					}
+				} else if (WidgetInfo.SHOP_ITEMS_CONTAINER.getGroupId() == widgetGroup) {
+					Widget[] children = widget.getDynamicChildren();
+					if (pendingExamine.getActionParam() < children.length) {
+						Widget widgetItem = children[pendingExamine.getActionParam()];
+						quantity = 1;
+						itemId = widgetItem.getItemId();
+					}
+				} else if (WidgetInfo.CLUE_SCROLL_REWARD_ITEM_CONTAINER.getGroupId() == widgetGroup) {
+					Widget[] children = widget.getDynamicChildren();
+					if (pendingExamine.getActionParam() < children.length) {
+						Widget widgetItem = children[pendingExamine.getActionParam()];
+						quantity = widgetItem.getItemQuantity();
+						itemId = widgetItem.getItemId();
+					}
 				}
 			}
-			else if (WidgetInfo.SHOP_ITEMS_CONTAINER.getGroupId() == widgetGroup)
+		}
+		else{
+			// We might be examining something in the open world.
+			if (pendingExamine.getType() == ExamineType.ITEM)
 			{
-				Widget[] children = widget.getDynamicChildren();
-				if (pendingExamine.getActionParam() < children.length)
-				{
-					Widget widgetItem = children[pendingExamine.getActionParam()];
-					quantity = 1;
-					itemId = widgetItem.getItemId();
-				}
+				quantity = 1;
+				itemId = pendingExamine.getId();
 			}
-			else if (WidgetInfo.CLUE_SCROLL_REWARD_ITEM_CONTAINER.getGroupId() == widgetGroup)
-			{
-				Widget[] children = widget.getDynamicChildren();
-				if (pendingExamine.getActionParam() < children.length)
-				{
-					Widget widgetItem = children[pendingExamine.getActionParam()];
-					quantity = widgetItem.getItemQuantity();
-					itemId = widgetItem.getItemId();
-				}
+			else {
+				return;
```

Comment:
This does not achieve anything; there is a guard clause on `itemId == -1` just after this anyway.